test/stateful_dataloader/test_state_dict.py

```diff
@@ -426,6 +428,32 @@ def load_state_dict(self, state):
         self.epoch = state["epoch"]
 
 
+class StateType(Enum):
+    INTEGER = (1,)
+    STRING = (2,)
+    TENSOR = (3,)
```

Comment:
I don't really care because this is just a test, but generally for python just use a string, I find this boilerplate just adds friction and doesn't usually help

Reply:
Oh I see from summary now

Reply:
```suggestion
```